MParT/ParameterizedFunctionBase.h

```diff
@@ -51,6 +51,8 @@ namespace mpart {
             @param coeffs A view containing the coefficients to copy.
         */
        virtual void SetCoeffs(Kokkos::View<const double*, MemorySpace> coeffs);
+       virtual void SetCoeffs(Kokkos::View<double*, MemorySpace> coeffs);
```

Comment:
This is a good idea, but I think you should just define it as such:
```cpp
void SetCoeffs(Kokkos::View<double*, MemorySpace> coeffs) {
    Kokkos::View<const double*, MemorySpace> const_coeffs;
    SetCoeffs(const_coeffs);
}
```
The differences here:
- It is not a virtual function
- It allows us to ensure only one `SetCoeffs` needs to be implemented in child classes